signer/xpi/README.rst

```diff
@@ -33,17 +34,57 @@ Each signer must have a type, a mode and the certificate and private key of
 an intermediate CA issued by either the staging or root PKIs of AMO (refer to
 internal documentation to issue those, as they require access to private HSMs).
 
-When a signature is requested, autograph will generate a private key and issue
-an end-entity certificate specifically for the signature request. The certificate
-is signed by the configured intermediate CA. The private key is thrown away
-right after the signature is issued.
+When a signature is requested, autograph will generate a private key
+and issue an end-entity certificate specifically for the signature
+request. The certificate is signed by the configured intermediate
+CA. The private key is thrown away right after the signature is
+issued. Note that if one XPI signer is including a recommendations
+file the other XPI signers should reserve that file path.
```

Comment:
Why? Are you concerned about signers overriding each other's recommendations, or something else?

Reply:
If we don't filter out the recommendations file someone could use a signer in mode "addon" and effectively get a recommended addon since it's the same OU.

addons-server should also filter out the file, but autograph shouldn't rely on that control.